packages/Statie/templates/statie-website/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0.0",
```

Comment:
I think `version` is not needed, but I'd add [`private: true`](https://docs.npmjs.com/files/package.json#private) property.